AndroidTest/app/build.gradle

```diff
@@ -105,8 +102,23 @@ repositories {
 task(uploadFixtures,type:Exec) {
     // need to upload the fixtures to the device/emulator external storage
     // this will work irrespective of current working directory due to use of $scriptLocation
-    def android = "$System.env.ANDROID_HOME"
-    commandLine "$android/platform-tools/adb","push", "-p", "$scriptLocation/../../fixture", "/sdcard/fixture"
+
+    def rootDir = project.rootDir
+    def localProperties = new File(rootDir, "local.properties")
+    def androidPath = null;
+    if (localProperties.exists()) {
+        Properties properties = new Properties()
+        localProperties.withInputStream { instr ->
+            properties.load(instr)
+        }
+        androidPath = properties.getProperty('sdk.dir')
+    }
+
+    if(androidPath == null){
+         androidPath = System.env.ANDROID_HOME
```

Comment:
Maybe check if the path in the env is also null and bomb out with an appropriate message